packages/components/src/components/hds/modal/index.ts

```diff
@@ -30,7 +70,7 @@ export default class HdsModalIndexComponent extends Component {
    * @default 'medium'
    */
   get size() {
```

Comment:
nit: we settled on explicitly defining return types for functions
```suggestion
  get size(): HdsModalSizes {
```